e2e/web/specs/play-sudoku.spec.ts

```diff
@@ -15,12 +15,20 @@ import { HeaderComponent } from "../components/header.component";
 import { StatisticsPage } from "../page/statistics.page";
 
 test.describe("pause", () => {
-  test("pause button", async ({ resumeGame }) => {
-    const sudokuBoard = new SudokuBoardComponent(resumeGame);
-    await sudokuBoard.pause.click();
-    const playPage = new PlayPage(resumeGame);
-    await playPage.playPageIsRendered();
-  });
+  const keys = ["button", "p", "P"];
+  for (const key of keys) {
+    test("pause button: " + key, async ({ resumeGame }) => {
```

Comment:
I am having test failures on windows because it creates folders based on the test names and "P" vs "p" difference is too similar I think. I posted example below of something that worked on my machine:
```typescript
test("pause button: " + key + " " + keys.indexOf(key) + "/" + keys.length, async ({ resumeGame }) => {
```

Reply:
Anything that would make the test names more unique would work

Reply:
Windows case insensitivity continuing to cause issues, shakes fist at sky lol. Of course they'll never fix it cause too many programs rely on things being sometimes cApitALIizED.